kuafu/pdfworker.py

```diff
@@ -19,9 +19,8 @@
 if PDF_BACKEND == 'PDFIUM':
     # follow https://github.com/BlockSigner/wowpng, and https://github.com/bblanchon/pdfium-binaries
     import ctypes
-    import pypdfium as PDFIUM
-    PDFIUM.FPDF_InitLibraryWithConfig(PDFIUM.FPDF_LIBRARY_CONFIG(2, None, None, 0))
-
+    import pypdfium2 as PDFIUM
```

Comment:
Once pypdfium2 v4 gets released, this would need to be import `pypdfium2.raw as PDFIUM`